src/Servers/HttpSys/src/HttpSysOptions.cs

```diff
@@ -219,6 +219,16 @@ public Http503VerbosityLevel Http503Verbosity
             }
         }
 
+        /// <summary>
+        /// Inline request processing instead of dispatching to the threadpool.
+        /// </summary>
+        /// <remarks>
+        /// This will run application code on the IO thread which is why this is unsafe.
+        /// This setting can make performance worse if there is expensive work that will end up holding onto the IO thread for longer than needed.
+        /// Test to make sure this setting helps performance.
```

Comment:
```suggestion
        /// Enabling this setting will run application code on the IO thread to reduce request processing latency. 
        /// However, this will limit parallel request processing to <see cref="MaxAccepts"/>. This setting can make 
        /// overall throughput worse if requests take long to process.
```

Reply:
@Tratcher will update. Also do I need to update docs anywhere? If so any links to info on that process?

Reply:
API docs will pull directly from here. I don't know if we're concerned about adding this to the [concept docs](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/httpsys?view=aspnetcore-5.0).